database/robustar-latest.sql

```diff
@@ -0,0 +1,37 @@
+--
```

Comment:
We don't need this SQL file at run time anymore. SQLAlchemy will internally expand python definition of entities (see in `back-end/database/model.py` and execute. However we can keep them for documentation purpose.